app/controllers/subscription_controller.rb

```diff
@@ -119,19 +119,20 @@ def digest
   end
 
   def multiple_add
-    unless params[:names]
+    unless params[:tagnames]
       flash[:notice] = "Please enter tags for subscription in the url."
       redirect_to "/subscriptions" + "?_=" + Time.now.to_i.to_s
+      return
```

Comment:
why do we require return here?

Reply:
Because the problem I was facing was, in an empty list, the unless condition was being entered but it wasn't able to redirect and give the required flash notice. With return, it was able to do the needful

Reply:
You mean if the user has not checked any checkbox. Am I right?
Now is it working?
We will require a separate test for this condition now. Thanks for being thorough.

Reply:
By test I mean Minitest.

Reply:
Apologies for misunderstanding.

Reply:
Let's address a new test for this condition in a later PR. It's a good idea.